kateryna-tolochko/README.md

```diff
@@ -0,0 +1,97 @@
+# Getting Started with AliExpress
+
+AliExpress is a popular online marketplace where you can find a wide variety of products at affordable prices. Whatever you are looking for, AliExpress offers a huge selection from sellers around the world. If you are new to AliExpress, this guide will help you get started and make the most of your shopping experience.
+
+## Step 1: Create an Account
+
+1. Visit the [AliExpress website](https://www.aliexpress.com/).
+
+2. Find **Join Free** button and click it.
+
+   ![ ](https://i.ibb.co/cyYxdZX/screen.png)
+
+3. After clicking, you should see a registration form and different options for signing up through social media accounts. Select your preferred method.
+
+4. If you chose to sign up through your social media accounts, you will be redirected to a page where you will need to give AliExpress permission to access your account information. Follow the on-screen instructions to complete the process.
```

Comment:
Steps #4 and #5 are derived from step #3, so it's better to make steps #3a and #3b. Also, there is a fork: you follow either step #4 or step #5, thus logically they can't be sequential. Consider rewriting.